logstash-plugins/logstash-integration-jdbc
# Vendor locally built Derby

## Release notes

Update Derby with locally built 10.15.2.1 version

## What does this PR do?


Adds the Gradle's task `deployLocallyDerbyArtifacts` (and its dependants) to checkout, through SVNKit, and build Derby artifacts for a specific release branch.
The build leverages Gradle's embedded `ant` and copy the interested artifacts (`derby` and `derbyclient`) into a local repository. 
The artifacts in local repository are generated each time the project is vendored, doing a checkout of a specific Subversion `revision` it's granted the reproducibility.
Updates also the `vendor` and `generateGemJarRequiresFile` to cope with the new locally shipped artifacts.

## Why is it important/What is the impact to the user?

This change permit to bundled latest versions of Derby which aren't officially released.

## Checklist




## Author's Checklist



## How to test this PR locally


- verify that the jars are built:
```
./gradlew deployLocallyDerbyArtifacts
```
- try the vendoring
```
./gradlew vendor
```

## Related issues


- Closes #149 

## Use cases



## Screenshots



## Logs
